README.md

```diff
@@ -94,12 +94,6 @@ Database will get initiated when you run `docker-compose up` for the first time
 
 You don't need to migrate database manually since migrations are executed every time `docker-compose up` is executed.
```

Comment:
That one has a hatch as technically not every `docker-compose up` starts the application and triggers migration. Maybe we could also replace `docker-compose up` with something that says that it happens on application start time.

Reply:
Absolutely, I have changed the wording.